src/move/data/dataloaders.py

```diff
@@ -65,6 +108,18 @@ def concat_cat_list(
 def concat_con_list(
     con_list: list[FloatArray],
 ) -> tuple[list[int], BoolArray, FloatArray]:
+    """
+    Concatenate a list of continuous data
+
+    Args:
+        con_list (list[FloatArray]): list with each continuous class data
+
+    Returns:
+        (tuple): a tuple containing:
```

Comment:
Gets formatted as a definition list by Sphinx, so perhaps it is better to use a list format instead. Example:

```
    Returns:
        : Tuple containing:

        - List of categorical data classes shapes (``num_samples`` x
          ``num_features`` x ``num_categories``)
        - Mask for patients with no measurments
        - 2D array of concatenated patients categorical data
```

Or plain text.